doc/man3/ENGINE_add.pod

```diff
@@ -585,9 +567,8 @@ L<OPENSSL_init_crypto(3)>, L<RSA_new_method(3)>, L<DSA_new(3)>, L<DH_new(3)>, L<
 
 =head1 HISTORY
 
-ENGINE_cleanup(), ENGINE_load_openssl(), ENGINE_load_dynamic(), and
-ENGINE_load_cryptodev() were deprecated in OpenSSL 1.1.0 by
-OPENSSL_init_crypto().
+ENGINE_cleanup() was deprecated in OpenSSL 1.1.0 by
+OPENSSL_init_crypto() and should not be used.
```

Comment:
I disagree, I think the fact that the routine says init does not imply that.  But I admit that perhaps I am too close to this, having written that phrase over a dozen times :)  But I pushed a new commit, see if that's okay.